UltimateTeam.Toolkit/Models/IListResponse.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+
+namespace UltimateTeam.Toolkit.Models
+{
+    public interface IListResponse
```

Comment:
Hi Trydis...
In my BOT I buy several players, some by normal bid others by buy now.
After the round of these bids I call a method (SendDuplicatedItemToClub) that receives this interface and sends all the duplicate players in the list to my club (I keep it just in case one day needs for some SBC for example). 

I created the interface only to need to implement the method only once since both of these types have this list of "duplicates". It really is not mandatory in case you want to retire feel free.